lib/puppet/provider/sensuctl.rb

```diff
@@ -99,7 +99,7 @@ def sensuctl_delete(*args)
     self.class.sensuctl_delete(*args)
   end
 
-  def self.sensuctl_set(command, name, property, value: nil, flags:  nil)
+  def self.sensuctl_set(command, name, property, value = nil, flags =  nil)
```

Comment:
Could you please add a comment that links to this PR letting folks know that puppetserver 5 uses jruby 1.9.3 or whatever and so we need it this way

Reply:
I checked and we're not actually using this function. We may have used it early on before all resources supported creation through JSON files. Should I just delete this function?

Reply:
There are two functions in `sensuctl.rb` that were helper functions no longer used. Deleting them wouldn't break anything.

Reply:
If we can delete them, even better

Reply:
Updated PR with functions removed.